deploy/osd-26951-camo-dance/config,yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Looks it's a typo for the `config.yaml` with an unexpected comma?

Reply:
thanks... fixed the typo